xwiki-platform-core/xwiki-platform-scheduler/xwiki-platform-scheduler-ui/src/main/resources/XWiki/SchedulerJobSheet.xml

```diff
@@ -40,49 +40,49 @@
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
   <content>{{velocity}}
-#if($doc.getObject('XWiki.SchedulerJobClass'))
-#set($job = $doc.getObject("XWiki.SchedulerJobClass"))
-(%class="xform"%)(((
-; **$services.localization.render('xe.scheduler.job.name')**
+#if ($doc.getObject('XWiki.SchedulerJobClass'))
+  #set ($job = $doc.getObject("XWiki.SchedulerJobClass"))
+  #macro (displayJobProperty $obj $propName)
+    #set ($class = $obj.xWikiClass)
+    #set ($prettyName = $class.get($propName).translatedPrettyName)
+    ; #if ($xcontext.action == 'edit')
+      {{html}}&lt;label for="${class.getName()}_${obj.number}_${propName}"&gt;$prettyName&lt;/label&gt;{{/html}}
+    #else
+      $prettyName
+    #end
+    : $doc.display($propName, $obj)
```

Comment:
maybe a `#define` could be enough? See http://velocity.apache.org/engine/1.7/user-guide.html#define

Reply:
@vmassol Not sure what you mean by that. Are you suggesting to use a #define instead of a #macro? Could you please elaborate?

Reply:
> Are you suggesting to use a #define instead of a #macro? 

Yes that was the idea. You'll need to decide what's best (macro or define). I haven't really looked in details, was just doing a suggestion since from the outside it seemed it was about computing something to display.